Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -642,14 +640,23 @@ static wxColour blend50(const wxColour& c1, const wxColour& c2)
   return a << 24 | b << 16 | g << 8 | r;
 }
 
-void CGameListCtrl::SetBackgroundColor()
+static wxColour ContrastText(const wxColour& bgc)
+{
+  return (((0.299 * bgc.Red()) + ((0.587 * bgc.Green()) + (0.114 * bgc.Blue())))) > 186 ? *wxBLACK : *wxWHITE;
```

Comment:
The arithmetic expression should likely be separate from the ternary conditional for readability. `186` should probably be a named constant as well.